examples/stocks/lib/i18n/regenerate.md

```diff
@@ -1,33 +1,26 @@
-## Regenerating the i18n files
+# Regenerating the i18n files
 
-The files in this directory are based on ../lib/stock_strings.dart
+The files in this directory are based on ../lib/i18n/stock_strings.dart,
```

Comment:
The files in this directory are used to generate stock_strings.dart, which is used by the stocks application to look up localized message strings.